templates/web/base/report/form/user_loggedout_by_email_password.html

```diff
@@ -0,0 +1,14 @@
+<label class="hidden-js hidden-nojs js-new-report-forgotten-shown" for="password_register">[% loc('New password:') %]</label>
+<label for="password_register" class="js-new-report-forgotten-hidden">[% loc('Password (optional)') %]</label>
+[% IF field_errors.password_register %]
+<p class='form-error'>[% field_errors.password_register %]</p>
+[% END %]
+<div class="general-notes js-new-report-forgotten-hidden" id="password_register_hint">
+  [% IF c.cobrand.moniker == 'bromley' %]
```

Comment:
I know this was copied from the other file, but could we potentially have a Bromley-specific version of this template, now that it's a smaller template? Would make the base template a bit cleaner.

Reply:
Yep, will do